.github/workflows/pr.yml

```diff
@@ -81,7 +81,7 @@ jobs:
         uses: peaceiris/actions-gh-pages@v3
         # if: github.ref == 'refs/heads/main'
         with:
-          github_token: ${{ secrets.RHDH_BOT_TOKEN }}
+          deploy_key: ${{ secrets.RHDH_BOT_TOKEN }}
```

Comment:
we can try this if you want -- but I'm not sure if we're using https or ssh to check out sources.

https://github.com/peaceiris/actions-gh-pages?tab=readme-ov-file#supported-tokens

Reply:
Ah, I see. Is there a way to check?

Reply:
try it and see if it breaks; then revert if so :D